confignode/src/main/java/org/apache/iotdb/confignode/client/async/handlers/AbstractRetryHandler.java

```diff
@@ -27,15 +27,15 @@
 
 public abstract class AbstractRetryHandler {
 
-  protected final int index;
-
   protected CountDownLatch countDownLatch;
+
   /**
-   * Map<Index, TDataNodeLocation> The DataNode that successfully execute the request will be
-   * removed from this map
+   * The DataNode that successfully execute the request will be removed from this list Because
+   * different requests will be sent to the same node when createRegions,so for CreateRegions ->
+   * Map<index, TDataNodeLocation> for others -> Map<dataNodeId, TDataNodeLocation>
```

Comment:
```suggestion
   * Map<DataNodeId, TDataNodeLocation> The DataNode that successfully execute the request 
   * will be remoed from this map.
```

We can keep this annotation remains the same since the CreateRegionHandler is a particular case. And we can move this part of special comment into the CreateRegionHandler